_includes/meta.html

```diff
@@ -61,6 +61,115 @@
     gtag('js', new Date());
 
     gtag('config', '{{ site.ga.gtag }}');
+
+    function checkDomainAvailability(requestedDomainId="domain-input", isDomainChoosing=false, alertId="usa-search--domain_message") {
```

Comment:
Optional/Strongly suggested: Much better but you CAN move this to a JS file even if it's loaded at the end of the file. Just wrap it in a document.loaded condition.